serverless/aws/examples/src/main/scalajs/sttp/tapir/serverless/aws/examples/LambdaApiJsExample.scala

```diff
@@ -0,0 +1,31 @@
+package sttp.tapir.serverless.aws.examples
+
+import cats.syntax.all._
+import sttp.monad.{FutureMonad, MonadError}
+import sttp.tapir._
+import sttp.tapir.server.ServerEndpoint
+import sttp.tapir.serverless.aws.lambda._
+import sttp.tapir.serverless.aws.lambda.js._
+
+import scala.scalajs.js.annotation._
+import scala.concurrent.{ExecutionContext, ExecutionContextExecutor, Future}
+
+object LambdaApiJsExample {
+
+  implicit val ec: ExecutionContextExecutor = ExecutionContext.global
+
+  val helloEndpoint: ServerEndpoint[Unit, Unit, String, Any, Future] = endpoint.get
+    .in("api" / "hello")
+    .out(stringBody)
+    .serverLogic { _ => Future(s"Hello!".asRight[Unit]) }
+
+  val options: AwsServerOptions[Future] = AwsFutureServerOptions.default.copy(encodeResponseBody = false)
+
+  val route: JsRoute[Future] = AwsFutureServerInterpreter(options).toRoute(helloEndpoint).toJsRoute
+
+  @JSExportTopLevel(name="handler")
+  def handler(event: AwsJsRequest, context: Any): scala.scalajs.js.Promise[AwsJsResponse] = {
+    import scala.scalajs.js.JSConverters._
+    route(event).toJSPromise
```

Comment:
maybe we could simplify this to sth like: 

```scala
@JSExportTopLevel(name="handler")
def handler(event: AwsJsRequest, context: Any) = AwsFutureServerInterpreter.handler(event, context, route)
```

where `route` is the `Route` returned by the interpreter. Or even better, maybe there can be an overload which accepts `ServerEndpoint`s directly and performs the interpretation & conversion?

Reply:
I moved it to `AwsJsRouteHandler`
```scala
  @JSExportTopLevel(name="handler")
  def handler(event: AwsJsRequest, context: Any): scala.scalajs.js.Promise[AwsJsResponse] = AwsJsRouteHandler.handler(event, route)
```

Reply:
what if we want to have a cats-effect JS interpreter? :)

Reply:
added handler for cats IO